.circleci/config.yml

```diff
@@ -367,6 +362,9 @@ workflows:
       - api_test
       - freezed
       - dry-run-release: *release-branches
+      - ios-integration-test
+      - macos-integration-test
+      - android-integration-test-build
```

Comment:
We can alternatively run these when merging on `main` so they don't run on every commit on every PR , what do you think?